### DISCUSSION TITLE
Accurate Data Retrieval Using Message Path Syntax in Foxglove Extension Panel Development

### DISCUSSION THREAD

**Discussion by Kezunlong1025** [foxglove/community#1037] [Q&A]
I am currently developing an extension panel for Foxglove, and I have utilized the Message path syntax. But how can I accurately retrieve the specific value of the variable that my Message path syntax points to within this extension panel? In other words, I have indicated the path, but how do I access the concrete data within the extension panel?

**Comment by jtbandes** [comment]
Duplicate of https://github.com/orgs/foxglove/discussions/1033
